ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -131,7 +129,7 @@ TorchCore::setAccelerator (const char *accelerators)
   accelerator = parse_accl_hw (accelerators, torch_accl_support);
   if (accelerator == ACCL_NONE)
     goto use_gpu_ini;
-  if ((accelerator & (ACCL_CPU | ACCL_DEFAULT)) != 0)
+  if ((accelerator & (ACCL_CPU)) != 0)
```

Comment:
What if ```accelerator == 0x3000``` ?

Reply:
I am not sure if I understand the question.
`parse_accl_hw` will return a accelerator which will ACCL_NONE or one of the accelerators supported by the framework (ACCL_CPU or ACCL_GPU) in this case.

Reply:
If user gives ```0x3000``` (CPU | GPU), pytorch may use either CPU or GPU. Thus, disabling GPU and enabling CPU only if 0x1000 (CPU bit) is enabled, is not "incorrect" logically, but, would not meet users' expectation normally. Users will expect (and understand) that it will use GPU if GPU is enabled if it's ```0x3000``` (CPU | GPU)

Reply:
Ah... parse_accl_hw_util will give a single element. OK.

Reply:
> uld not meet users' expectation normally. Users will expect (and understand) that it will use GPU if GPU is enabled if it's 0x3000 (CPU | GPU

On the side note, user is supposed to give `CPU, GPU` than `CPU | GPU`.